package.json

```diff
@@ -18,7 +18,6 @@
   },
   "devDependencies": {
     "@taplo/cli": "^0.5.2",
-    "@types/react": "18.2.0",
```

Comment:
Intentional! This doesn't do anything to make the repo function correctly, but it can mess up your editor's understanding of React. 😄